proto/answer.proto

```diff
@@ -78,17 +52,29 @@ message ReadableConceptTree {
             repeated Node list = 1;
         }
 
-        message ReadableConcept {
-            oneof readable_concept {
-                EntityType entity_type = 1;
-                RelationType relation_type = 2;
-                AttributeType attribute_type = 3;
+        message Leaf {
+            oneof leaf {
+                Empty empty = 1;
```

Comment:
New field - nullable entries (` fetch { $x.name }`, where `name` is `card(0.1)` ).